src/checkqueue.h

```diff
@@ -132,16 +139,31 @@ class CCheckQueue
     {
     }
 
+    //! Create a pool of new worker threads.
+    void StartWorkerThreads(const int threads_num)
+    {
+        if (threads_num <= 0) return;
+        assert(m_worker_threads.empty());
+        m_stopped = false;
+        m_worker_threads.reserve(threads_num);
+        for (auto n = 0; n < threads_num; ++n) {
```

Comment:
```suggestion
        for (int n = 0; n < threads_num; ++n) {
```

is shorter and more precise

Reply:
Agree.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18710#issuecomment-619008172).